tests/WP_Ultimo/Gateways/Stripe_Connect_Application_Fee_Test.php

```
@@ -0,0 +1,389 @@
+	 */
+	public function test_settings_includes_application_fee_field() {
+		// The settings method should run without errors
+		$this->gateway->settings();
+		
+		// Test that the method exists and can be called
+		$this->assertTrue(method_exists($this->gateway, 'settings'));
+		
+		// We can't test the specific registration of settings in this test environment,
+		// but we can verify the method implementation includes the field
+		$this->assertStringContainsString('application_fee_percentage', 
+			$reflection = new \ReflectionClass($this->gateway));
```

PHPMD (2.15.0)

Avoid unused local variables such as '$reflection'. (undefined)

(UnusedLocalVariable)